suite-common/wallet-types/src/account.ts

```diff
@@ -45,6 +45,13 @@ type AccountNetworkSpecific =
           page: AccountInfo['page'];
       };
 
+export interface AccountLastKnownState {
```

Comment:
or should we use `Account.misc` for that?

Reply:
I don't think it's that misc. And to what `networkType` would you add it? I think that it makes sense either on all accounts or only the coinjoin account specifically.